[2024-11-06T06:21:34.626+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-06T06:21:34.754+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag.database_tasks.check_integrity manual__2024-11-06T06:21:25.886567+00:00 [queued]>
[2024-11-06T06:21:34.763+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag.database_tasks.check_integrity manual__2024-11-06T06:21:25.886567+00:00 [queued]>
[2024-11-06T06:21:34.763+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-06T06:21:34.781+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): database_tasks.check_integrity> on 2024-11-06 06:21:25.886567+00:00
[2024-11-06T06:21:34.784+0000] {standard_task_runner.py:72} INFO - Started process 1315 to run task
[2024-11-06T06:21:34.787+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'database_tasks.check_integrity', 'manual__2024-11-06T06:21:25.886567+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpa00yi_nk']
[2024-11-06T06:21:34.789+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask database_tasks.check_integrity
[2024-11-06T06:21:34.844+0000] {task_command.py:467} INFO - Running <TaskInstance: test_dag.database_tasks.check_integrity manual__2024-11-06T06:21:25.886567+00:00 [running]> on host 8a2ac309b35c
[2024-11-06T06:21:34.922+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='veil' AIRFLOW_CTX_DAG_ID='test_dag' AIRFLOW_CTX_TASK_ID='database_tasks.check_integrity' AIRFLOW_CTX_EXECUTION_DATE='2024-11-06T06:21:25.886567+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-06T06:21:25.886567+00:00'
[2024-11-06T06:21:34.923+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-06T06:21:34.940+0000] {logging_mixin.py:190} INFO - Connection to PostgreSQL DB successful
[2024-11-06T06:21:34.940+0000] {logging_mixin.py:190} INFO - /app/***/dags/database/integrity_check.py
[2024-11-06T06:21:35.076+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-06T06:21:35.089+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-06T06:21:35.089+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=database_tasks.check_integrity, run_id=manual__2024-11-06T06:21:25.886567+00:00, execution_date=20241106T062125, start_date=20241106T062134, end_date=20241106T062135
[2024-11-06T06:21:35.131+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-06T06:21:35.145+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-06T06:21:35.148+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
